doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,123 @@
+[id='openshift-springboot-proc_{context}']
+= Running a SpringBoot business application on {OPENSHIFT}
+
+To run your {PRODUCT} SpringBoot business application on {OPENSHIFT}, create an immutable image and push this image to your {OPENSHIFT} environment.
+
+ifdef::PAM[]
+Optionally, you can also use {CENTRAL} Monitoring to monitor the execution of business processes in your application.
+endif::PAM[]
+
+.Prerequisites
+
```

Comment:
Same as previous review on smaller doc. You should be using "You have.." for the prereqs. Something that you have done is written in the present perfect style.